components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -98,6 +98,15 @@ export default function OrganizationSelector() {
             }
             // Show billing if user is an owner of current org
             if (isOwner) {
+                // Add Admin link for owners
+                linkEntries.push({
+                    title: "Admin",
+                    customContent: <LinkEntry>Admin</LinkEntry>,
+                    active: false,
+                    separator: false,
+                    link: "/org-admin",
+                });
```

Comment:
🤙

Reply:
We should probably hide this for non-dedicated. Does not harm on gitpod.io, but also does not make sense. :+1: